tests/acceptance/features/coreApiTrashbin/trashbinDelete.feature

```diff
@@ -189,7 +189,7 @@ Feature: files and folders can be deleted from the trashbin
     And as "Alice" the file with original path "/PARENT/CHILD/child.txt" should not exist in the trashbin
     And as "Alice" the folder with original path "/PARENT/CHILD/" should not exist in the trashbin
     But as "Alice" the file with original path "/PARENT/parent.txt" should exist in the trashbin
-    But as "Alice" the file with original path "/FOLDER/parent.txt" should exist in the trashbin
+    And as "Alice" the file with original path "/FOLDER/parent.txt" should exist in the trashbin
```

Comment:
In this case, having `But` in sequential lines is not correct. So the rule could be that there can be multiple `But` in a `Then` section, but they cannot be sequential.

Maybe there will be scenarios where `But` could happen sequentially - but I think that it would be strange, and probably the `Then` steps can always be rearranged to avoid it.

Reply:
Having repetitive steps will give an error.